datasets/aime-2024/prepare.py

```diff
@@ -0,0 +1,147 @@
+import json
```

Comment:
Let's create a new requirements.txt inside datasets folder and put all necessary packages there and add it to the .github/workflow/tests.yml to both unit and cluster tests as a separate installation. Let's also add it to docs/prerequisites.md to say that to get all datasets you need to install those requirements both on cluster and locally. Besides requests/bs4 please also add pyyaml and `evalplus @ git+https://github.com/evalplus/evalplus@c91370f` to that req file

Reply:
added